.github/workflows/terraform_plan.yaml

```diff
@@ -9,15 +9,16 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - name: Plan production
-      if: success() && contains(github.base_ref, 'master')
```

Comment:
may have to double-check this. `base_ref` of a pull request trigger will point to the target branch where we run the plan but `ref_name` will point to the "feature branch".

Reply:
Ah i misread, i see you're using `github.ref`. Still would be good to clarify why this change was needed since `ref` looks less straightforward than`base_ref`

Reply:
Thank you for catching that! Changed to use == base_ref